runtimes/datascience/ubi9-python-3.11/Pipfile

```diff
@@ -7,41 +7,44 @@ name = "pypi"
 
 [packages]
 # Datascience and useful extensions
-boto3 = "~=1.34.50"
-kafka-python = "~=2.0.2"
-matplotlib = "~=3.8.3"
-numpy = "~=1.26.4"
-pandas = "~=2.2.0"
-plotly = "~=5.20.0"
-scikit-learn = "~=1.4.0"
-scipy = "~=1.12.0"
-skl2onnx = "~=1.16.0"
-codeflare-sdk = "~=0.19.1"
-memray = "~=1.10.0"  # any version above this would cause issue with rich package
+boto3 = "~=1.35.14"
+kafka-python-ng = "~=2.2.2"
+kfp = "~=2.9.0"
```

Comment:
the kfp , package is not needed in the runtime

Reply:
Great, I removed it.